includes/frontend.php

```diff
@@ -638,7 +638,7 @@ public function enqueue_styles() {
 
 			wp_enqueue_style( 'elementor-frontend' );
 
-			wp_enqueue_style( 'swiper' );
+			//wp_enqueue_style( 'swiper' );  We can remove this in version 3.26
```

Comment:
Please add this comment to the ED-15471 task.

```suggestion
			// TODO: Remove in version 3.26.0 [ED-15471]
			//wp_enqueue_style( 'swiper' );
```

Reply:
Can you implement this suggestion?